### ISSUE TITLE
Annotations not showing in core.tab

### ISSUE THREAD

**Issue by sabrinadiemert** [tseemann/snippy#254]
Hi @tseemann,

With the update to snippy v.4, are annotations dropped from the core.tab files produced in snippy-core? I can see annotations in my individual snp.tab files (i.e. for each sample), but after I run snippy-core on 21 samples, core.tab only has headings CHR, POS, REF, and all my samples names. 

Thanks!

**Comment by tseemann** [comment]
Yes, snippy 4 does not yet include the annotations in the .tab file.

One could use `bedtools intersect` with your `snps.vcf` and `references.gff` to achieve it, and I am looking to do it that way in the future.

**Comment by sabrinadiemert** [comment]
Thanks for confirming @tseemann. I ended up annotating it semi-manually, with a python script to compare with & fill in annotations from individual isolate snp.tab files. I'll try playing around with bedtools next time though. Cheers!

**Comment by flyant2** [comment]
@sabrinadiemert :
snippy-core(4.0-dev2) can provide annotations.

**Comment by mabouelk** [comment]
Hi All, 

I have the same issue If there anyone can help me?

**Comment by tseemann** [comment]
It's not really possible to offer true annotations because a position can be the same as REF, or different from REF in many different way. ie. if REF=A, then different samples could be T,C,G or even N.  The 'snpEff' annotation will NOT be the same.

If you just want to know what gene it occured in, then that could be possible.

This will also work if you spend some time on it:
https://github.com/tseemann/snippy/issues/254#issuecomment-472624345
